federated-learning/CLOUD.md

```diff
@@ -0,0 +1,59 @@
+# Federated Learning on Google Cloud Platform
+
+This guide will show you how to launch Google Cloud instances to run the federated learning tutorial in the cloud.
+
+If you want to run this part of the tutorial, you will need a [GCP account](https://cloud.google.com/) and to install [Google SDK](https://cloud.google.com/sdk/install).
+
```

Comment:
We should also clarify that they will have create a gcp project and set the project in the terminal. They probably can do everything with `cli`: https://cloud.google.com/resource-manager/docs/creating-managing-projects

Just thinking about people who have never used gcp before.

Reply:
Hmm this page tells them to make a project https://cloud.google.com/sdk/install I think once they do that and authenticate with gcloud that it will automatically use that new project.

Reply:
Sounds good!